app/models/courses_users.rb

```diff
@@ -86,6 +86,10 @@ def live_revisions
     course.revisions.joins(:article).where(user_id: user_id).live
   end
 
+  def total_uploads
+    course.uploads.where(user_id: user.id).count
```

Comment:
Doing it this way will result in one extra database query for each user, whenever the users.json for a course is requested.

To avoid the performance impact, the next steps should be:

1. Generate a migration to add `total_uploads` to the `courses_users` database table.
2. Move this query into the update_cache method, so that instead of doing the query on each read, it gets done once and stored as an attribute along with the other stats.
3. Check whether all of the database indexes to make the query as performant as possible are already present, and add them if necessary.